core/common/src/format/DateTimeFormat.kt

```diff
@@ -15,25 +15,33 @@ import kotlinx.datetime.internal.format.parser.*
 public sealed interface DateTimeFormat<T> {
```

Comment:
Suggestion: consider mentioning that by convention, one can see the available formats for X via `X.Formats.*`